richtextfx/src/main/java/org/fxmisc/richtext/model/TextEditingArea.java

```diff
@@ -197,4 +265,39 @@ default void replace(IndexRange range, StyledDocument<PS, S> replacement) {
      */
     @Deprecated
     public void positionCaret(int pos);
+
+    /**
+     * Returns the absolute position (e.g. the spot in-between characters) to the left of the given column in the given paragraph.
```

Comment:
I used "i.e." a lot until I saw you use "e.g." a lot. So, I thought they were the same and just used that whenever it related to you or this project. :smiley: Having looked it up, I see the difference now.